src/metabase/public_settings.clj

```diff
@@ -9,112 +9,113 @@
             [metabase.public-settings.metastore :as metastore]
             [metabase.util.i18n :refer [available-locales-with-names set-locale]]
             [metabase.util.password :as password]
+            [puppetlabs.i18n.core :as i18n :refer [tru]]
             [toucan.db :as db])
   (:import java.util.Locale
            java.util.TimeZone))
 
 (defsetting check-for-updates
-  "Identify when new versions of Metabase are available."
+  (tru "Identify when new versions of Metabase are available.")
   :type    :boolean
   :default true)
 
 (defsetting version-info
-  "Information about available versions of Metabase."
+  (tru "Information about available versions of Metabase.")
   :type    :json
   :default {})
 
 (defsetting site-name
-  "The name used for this instance of Metabase."
+  (tru "The name used for this instance of Metabase.")
   :default "Metabase")
 
 ;; This value is *guaranteed* to never have a trailing slash :D
 ;; It will also prepend `http://` to the URL if there's not protocol when it comes in
 (defsetting site-url
-  "The base URL of this Metabase instance, e.g. \"http://metabase.my-company.com\"."
+  (tru "The base URL of this Metabase instance, e.g. \"http://metabase.my-company.com\".")
   :setter (fn [new-value]
             (setting/set-string! :site-url (when new-value
                                              (cond->> (s/replace new-value #"/$" "")
                                                (not (s/starts-with? new-value "http")) (str "http://"))))))
 
 (defsetting site-locale
-  "The default language for this Metabase instance. This only applies to emails, Pulses, etc. Users' browsers will
-   specify the language used in the user interface."
+  (tru "The default language for this Metabase instance. This only applies to emails, Pulses, etc. Users' browsers will
```

Comment:
Gettext can't follow line breaks 😿. In this case, it would be easy to rewrite it like this:
```
(str (tru "The default language for this Metabase instance.")
     (tru "This only applies to emails, Pulses, etc. Users' browsers will specify the language used in the user interface.")
```
That also has the added benefit of creating two internationalized sentences. If one of the sentences shows up somewhere else, we'll only need to translate it once.